src/Components/Operator.js

```diff
@@ -1,10 +1,15 @@
+import React from 'react';
 import { useState } from 'react';
 
```

Comment:
- [ ] Your calculator is not able to successfully perform basic calculator operations. When an operation is performed, for instance, _1X6_ and then the _=_ button is clicked no answer is displayed. Kindly ensure to improve upon this aspect.


**Operation error:**
![image](https://user-images.githubusercontent.com/12745474/152821047-22f2549f-b85b-459f-b4f4-f007f5d4c814.png)